README.md

```diff
@@ -181,6 +184,82 @@ Events caused by pointer movement or scrolling do not count as "input" for the
 purpose of the recent input exclusion and the input-related attributes on the
 `LayoutShift` entry.
 
+### Source Attribution
+
+_NOTE: The `sources` attribute is not yet implemented in any browser, and its
+definition is still under active development._
+
+On a complex website, it can be difficult to understand the cause of a high
+CLS score given only the numeric values in the `value` attribute of the
+`LayoutShift` entries.
+
+The `sources` attribute connects the `LayoutShift` back to the specific DOM
+elements that experienced the shift, so that the developer has more insight
+into the causes of layout instability on their site.
+
+The `sources` attribute is an array of up to 5 `LayoutShiftAttribution` objects:
+
+```idl
+interface LayoutShiftAttribution {
+    Node node;
+    DOMRect previousRect;
+    DOMRect currentRect;
+};
+```
+
+Each attribution contains a reference to a [shifted](#Shifting-Nodes) DOM node
+along with rects that describe its visual representation in the viewport
+before and after the shift.
+
+#### Prioritization by Impact
+
+If more than 5 nodes have shifted in a single animation frame, the user agent
```

Comment:
The selection principles occur regardless of the number of shifted nodes in the frame, so this sentence is inaccurate.

Reply:
Good catch, the containment check may reduce the attribution even below 5.  Reworded.